api/parsers.go

```diff
@@ -119,7 +119,7 @@ func (p *Parsers) Get(reposistoryName string, parserName string) (*Parser, error
 				SourceCode string
 				TestData   []string
 				TagFields  []string
-			} `graphql:"parser(name: $parserName)"`
+			} `graphql:"parser(id: $parserName)"`
```

Comment:
Schema on master still has this:

```
 "Lookup a parser by name."
  parser(name: String!): Parser`
```

Reply:
Alright, my apologies. I'll update this correctly, once the other changes have reached master.